build/configs/artik053/STDK/defconfig

```diff
@@ -29,7 +29,6 @@ CONFIG_BUILD_FLAT=y
 #
```

Comment:
Please add commit description

Reply:
The rebase was updated forgot to continue the rebase.